maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/branch/JGitBranchCommand.java

```diff
@@ -65,9 +65,7 @@ protected ScmResult executeBranchCommand(
             throw new ScmException("This provider doesn't support branching subsets of a directory");
         }
 
-        Git git = null;
-        try {
-            git = JGitUtils.openRepo(fileSet.getBasedir());
+        try (Git git = JGitUtils.openRepo(fileSet.getBasedir())) {
```

Comment:
This actually introduced a regression, because closing `git` is not the same as calling `JGitUtils.closeRepo(git)`! Fixed in https://github.com/apache/maven-scm/pull/1271/files#diff-69ae8c5f67669f23b40308b3fb41313eeb78ac35a247a28de76b68f5b99de9e8R91.